pocs/scheduler/constraint.py

```diff
@@ -159,3 +159,23 @@ def get_score(self, time, observer, observation, **kwargs):
 
     def __str__(self):
         return "Moon Avoidance"
+
+class Seqencer(BaseConstraint):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_score(self, time, observer, observation, **kwargs):
+        veto = False
+        score = self._score
+
+        target = observation.field
+        observed = observation.observed_list
+
+        if target in observed:
+            veto = True
+
+    return veto, score * self.weight
+
+    def __str__(self):
+        return "Don't repeat observations"
```

Comment:
How this gets used it's better to actually just have something like "AlreadyVisited". An example of how it shows up is with the word `Altitude` here:

```
I0116 12:30:21.285 dispatch.py:56       Checking Constraint: Altitude
D0116 12:30:21.285 dispatch.py:59       	Observation: Kepler 1100
```